### PR TITLE
#51 Update docs for ohara 0.10.0 release

### PR DESCRIPTION
**Pull request by konekoya**
Work in progress :)

### CODE REVIEW

File: docs/src/quickstart_vm.rst [hunk 1]
```diff
@@ -152,8 +152,8 @@ After the installation is complete, you should see something like the following:
 
     > Ohara ready on http://192.168.56.105:5050
 
-As we can see in the above, the VM's IP address is `192.168.56.105` (this address can be varied).
-We can then open the browser and enter this URL in browser's address bar `http://192.168.56.105:5050`. to open Ohara manager (Ohara's UI, we will introduce it in the following section).
+As we can see in the above, the VM's IP address is ``192.168.56.105`` (this address will be varied depending on your VirtualBox network settings).
+We can then open the browser and enter this URL in browser's address bar ``http://192.168.56.105:5050``. to open Ohara manager (Ohara's UI, we will introduce it in the following section).
```
Comment (alan791205) [inline docs/src/quickstart_vm.rst:156]: ```address bar ``http://192.168.56.105:5050``. to``` 多個點?

Comment (konekoya) [reply docs/src/quickstart_vm.rst:156]: Yep, will drop it :)

File: docs/src/quickstart_vm.rst [hunk 2]
```diff
@@ -167,22 +167,22 @@ Node
  Node is the basic unit of running service. It can be either a physical machine or VM.
 
 Workspace
-  Workspace contains multiple Ohara services including: Zookeepers, Brokers and Workers. And pipelines are service that run in a workspace
+  Workspace contains multiple Ohara services including: Zookeepers, Brokers and Workers. And pipelines are services that run in a workspace
```
Comment (konekoya) [inline docs/src/quickstart_vm.rst:170]: @alan791205 👆 fixed

File: docs/src/quickstart_vm.rst [hunk 3]
```diff
@@ -167,22 +167,22 @@ Node
...
 
 Pipeline
   Pipeline allows you to define your data stream by utilizing **Connector** to connect to external storage systems,
   as well as a **Stream** to customize data transformation and stream processing.
 
 Connector
-  Connector is used to connect to the external storage systems. It has two types - source connector and sink connector.
-  Source connector pulls data from another system and then push to topic. By contrast, Sink connector pulls data from
-  topic and then push to another system.
+  Connector connects to external storage systems like Database, HDFS or FTP. It has two types - source connector and sink connector.
```
Comment (konekoya) [inline docs/src/quickstart_vm.rst:177]: @alan791205 I'm not sure if i should use singular or plural here... should it be: `Databases, HDFS or FTPs`?

**Comment by vitojeng** [comment]
Thanks @alan791205 for reviewing!

File: docs/src/quickstart_vm.rst (hunk 3 above)
Comment (alan791205) [reply docs/src/quickstart_vm.rst:188]: I think only ```Database``` has a plural form, and yes, it should be plural.

File: docs/src/quickstart_vm.rst [hunk 4]
```diff
@@ -275,6 +286,9 @@ Open Ohara Manager with your browser (http://192.168.56.105:5050) and should see
 .. tip::
   You can create more workspace with the plus (+) icon in the App bar.
 
+.. warning::
+  Please keep in mind that this is a quick start VM where we run everything on a single node with only 8GB of RAM and 2 CPU core. When Ohara is running without enough resources, it could be very unstable and causing unexpected errors. We would highly recommend you to add more RAM and CPU core to your VM If you plan to create more than one workspace or lots of pipelines, connectors, streams, etc.
+
```
Comment (konekoya) [inline docs/src/quickstart_vm.rst:291]: @vitojeng This tells users more RAM and CPU cores are required when they plan to create more workspaces and pipelines...

File: docs/src/quickstart_vm.rst [hunk 5]
```diff
@@ -531,3 +545,52 @@ We now provide a few debugging tools that can help you pin down unexpected error
 
   .. figure:: images/devtool-topic-log.png
     :alt: DevTool topic log
+
+.. _delete-workspace:
```
Comment (konekoya) [inline docs/src/quickstart_vm.rst:549]: Also added delete workspace and restart workspace

File: docs/src/quickstart_vm.rst [hunk 6]
```diff
@@ -152,8 +152,19 @@ After the installation is complete, you should see something like the following:
...
+
+ .. note::
+  After shutting down your VM, the docker containers will be deleted and restarted for your next login
+
+ .. tip::
+  These docker images will only be downloaded once, if you would like to re-download it again, you will need to manually remove these images:
+  
+  1. Remove running containers, in your command prompt: ``docker rm -vf $(docker ps -a -q)``
+  2. After all docker containers are properly stopped, delete all images with: ``docker rmi -f $(docker images -a -q)``
+  3. Restart your VM, the images should be pulled again from Docker Hub
```
Comment (konekoya) [inline docs/src/quickstart_vm.rst:167]: Tells users how to re-download images

**Comment by konekoya** [comment]
@vitojeng @alan791205 I think the PR is ready for final review :)

File: docs/src/quickstart_vm.rst (hunk 6 above)
Comment (vitojeng) [reply docs/src/quickstart_vm.rst:167]: This section maybe only for SNAPSHOT release version.
If a release version not a SNAPSHOT verison, the ohara image will not change anymore. So I guess user have no need to re-download images.

Comment (konekoya) [reply docs/src/quickstart_vm.rst:167]: You're right. This seems not necessary at all 🤔

File: docs/src/quickstart_vm.rst (hunk 5 above)
Comment (vitojeng) [reply docs/src/quickstart_vm.rst:543]: Cool~

File: docs/src/quickstart_vm.rst (hunk 4 above)
Comment (vitojeng) [reply docs/src/quickstart_vm.rst:285]: Ok, thanks!

**Comment by konekoya** [comment]
Thanks for the review everyone :)

**Pull request merged**